### DISCUSSION TITLE
[Feature Request] In the Power options in the Win10 Start Menu, change the icons so it uses "Segoe UI Fluent Icons" instead of "Segoe MDL2 Assets"

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#984] [General]
This will provide some consistency with Windows 11

Current (Metro Design Language)
![Screenshot 2022-03-03 191938](https://user-images.githubusercontent.com/76626288/156675313-649d2778-3fec-4db9-ac76-86fd8f481353.png)

Win11 (Fluent Design Language)
![Screenshot 2022-03-03 192218](https://user-images.githubusercontent.com/76626288/156675453-6bd7808c-58a3-436a-91cc-ec55a473234c.png)

**Comment by Shomnipotence** [comment]
hard in coding

**Comment by plofhaan** [comment]
I like the inarticulate design better.

**Comment by valinet** [comment]
I think the ground work is laid down: maybe this could be achieved by locating those buttons in the live visual tree and altering some image/icon property of them. I will leave this as a research task for the community, personally it doesn’t bother me, but if someone takes the time to sort this out, I will include it as an option. Likewise for system tray icons.
